invenio_jobs/administration/jobs.py

```diff
@@ -46,7 +46,8 @@ class JobsAdminMixin:
             "icon": "calendar",
         },
         "runs": {
-            "text": "Run now",
+            "text": "Configure and run",
```

Comment:
```suggestion
            "text": "Configure run",
```

Reply:
I don't think this indicates that this is where you would run the job.